README.md

```diff
@@ -5,6 +5,27 @@ A Slicer extension that enables user to interact with a Slicer scene using virtu
 
 ![](VR.png)
 
+Building steps
```

Comment:
Also add the following before the section `Building steps`:

This extension is current under development, waiting it is available in the Slicer Extension Manager, follow these instructions.